docker-compose.yaml

```diff
@@ -0,0 +1,53 @@
+version: '3.7'
+services:
+  wehe-server:
+    image: local-wehe-server
+    build:
+      context: .
+      dockerfile: Dockerfile
+    volumes:
+      - ./local:/local
```

Comment:
Okey, I figured out why I see no output file. We need to add this line to volumes (i.e., after line 9)

- ./data:/data